packages/shared/src/domain/contact/__tests__/email.test.ts

```diff
@@ -86,15 +145,25 @@ describe("email", () => {
       expect(normalizeEmailNewSafe(input)).toBe(expectedOutput);
     });
 
-    it("should relace mailto: prefix", () => {
+    it("should replace mailto: prefix", () => {
       const expectedOutput = exampleEmail;
       const input = "mailto:" + expectedOutput;
       expect(normalizeEmailNewSafe(input)).toBe(expectedOutput);
     });
 
-    it("should return undefined for emails that invalid", () => {
+    it("should return undefined for emails that are invalid", () => {
       const input = "this.is.not.an.email";
       expect(normalizeEmailNewSafe(input)).toBeUndefined();
     });
+
+    it("should handle emails with special characters", () => {
+      const input = "test!user@example.com";
+      expect(normalizeEmailNewSafe(input)).toBe("test!user@example.com");
+    });
+
+    it("should return undefined for phone number format", () => {
+      const input = "+1234567890@example.com";
+      expect(normalizeEmailNewSafe(input)).toBeUndefined();
```

Comment:
Actually, I just read this test so it seems like the `+1` prefix is the expected behavior - let me know if that's the case.